server/util/json-ld.js

```diff
@@ -51,6 +52,21 @@ export function museumLd(museum) {
   return objToJsonLd(newMuseum, 'Museum');
 }
 
+export function eventLd(event: Event) {
+  return objToJsonLd({
+    name: event.title,
+    // TODO: This is not always at Wellcome, but we don't collect that yet
+    location: {
+      '@type': 'Place',
+      name: 'Wellcome Collection',
+      address: objToJsonLd(wellcomeCollectionAddress, 'PostalAddress', false)
+    },
+    startDate: event.when.map(range => range.start),
+    endDate: event.when.map(range => range.start),
+    description: event.description
```

Comment:
worth stripping out the html tags? .replace(/(<([^>]+)>)/ig, '')

Reply:
Not sure - surely leaving links etc in is the thing to do?